### PR TITLE
Register daisuu.is-a.dev

### PR DESCRIPTION
**Pull request by DaisuKiee**
Register daisuu.is-a.dev with CNAME record pointing to c6571a19-e974-48d0-848f-fff7f971c3b3.id.repl.co.

### CODE REVIEW

**Comment by DaisuKiee** [comment]
how can i do this?

File: domains/daisukie.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "DaisuKiee",
+           "email": "vincentbernaberomeo@gmail.com",
+           "discord": "774974687394267157"
+        },
+    
+        "record": {
+            "A": ["68.65.123.44"]
```
Comment (CuteDog5695) [inline domains/daisukie.json:9]: 403 forbidden, please give a preview of your website.

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by triach-rold — COMMENTED** [review]
Website is unreachable.

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**